PWGHF/HFC/TableProducer/correlatorDplusHadrons.cxx

```diff
@@ -39,8 +39,7 @@ double getDeltaPhi(double phiD, double phiHadron)
 
 /// definition of variables for Dplus hadron pairs (in data-like, MC-reco and MC-kine tasks)
 const int npTBinsMassAndEfficiency = o2::analysis::hf_cuts_dplus_to_pi_k_pi::nBinsPt;
-const double efficiencyDmesonDefault[npTBinsMassAndEfficiency] = {};
-auto efficiencyDmeson_v = std::vector<double>{efficiencyDmesonDefault, efficiencyDmesonDefault + npTBinsMassAndEfficiency};
+std::vector<double> efficiencyDmeson;
```

Comment:
```suggestion
std::vector<double> efficiencyDmeson(npTBinsMassAndEfficiency);
```

Reply:
Thank you, I will do that way.

Reply:
@Simple-Shyam you can simply click on "Commit suggestion" and it will enter in your PR

Reply:
Thank you @fgrosa , I did it.